srcpkgs/erlang_ls/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'erlang_ls'
+pkgname=erlang_ls
+version=0.52.0
+revision=1
+build_style=gnu-makefile
+make_build_args="PREFIX=/usr"
+hostmakedepends="rebar3 erlang git"
```

Comment:
can it use chroot-git (`export PATH=/usr/libexec/chroot-git:$PATH`)?

Reply:
yes, it can
thanks